index.js

```diff
@@ -0,0 +1,14 @@
+const prompt = require("prompt-sync")()
```

Comment:
You can just use `window.prompt`, no need to require this package. And try to use import statement instead of require

Reply:
when using window.prompt it is throwing an error like window is not defined and actually i am using node so i think i don't need to use window but still throwing an error